recipes/statsmodels/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "statsmodels" %}
+{% set version = "0.8.0rc1" %}
+{% set build = 0 %}
+{% set md5 = "da32434ebfebae2c7506e9577ac558f5" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/c8/0a/71ea3dbc6fd712e18123a9e513066c8f5e19dbcabc49b5ba7ab07c97ea29/statsmodels-0.8.0rc1.tar.gz#md5=da32434ebfebae2c7506e9577ac558f5
+    md5: {{ md5 }}
+
+build:
+    number: {{ build }}
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - cython
+        - numpy
```

Comment:
Yep, it has `cimport numpy` in places. Using `x.x` for `numpy` in `build` and `run` is the right way to go here.